apps/cloudfront-functions/src/viewer-request-handler.ts

```diff
@@ -5,16 +5,20 @@
 /* eslint-disable functional/no-expression-statements */
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const handler = (
+export const handler = (
```

Comment:
You cannot use the `export` keyword here. Doing this, you are kind of reverting [this PR that fixed the issue](https://github.com/pagopa/b2b-portals/pull/125).

If you use the `export` there, you will get a runtime error on the function